### ISSUE TITLE
Uyuni List Page Size Default to 15 Items

### ISSUE THREAD

**Issue by fullerje** [uyuni-project/uyuni#7675]
### Problem description

Since the upgrade to 2023.09, the "Items Per Page" defaults to 15 and remains static.

![uyuni_systems](https://github.com/uyuni-project/uyuni/assets/125321411/09b6a5ff-c1cb-4d41-8f9c-afcb4fda1dff)

This is regardless of the value posted in My Preferences

![uyuni_preferences](https://github.com/uyuni-project/uyuni/assets/125321411/d968e3b8-a45d-4003-b5c7-c55930ec13e6)

A manual over-ride will temporarily work but will default back to 15 once refreshed or reloaded.   

### Steps to reproduce

1.   Log into UI
2.   Click on Systems Tab
3.   Items Per Page listed at 15. 
4.   Manually changed to 100 and restart UI. 
5.   Items Per Page remains at 15
6.   Manually setting value allows for the desired allotment of items per page but will revert back to 15 upon reload or refresh. 
...

### Uyuni version

```bash
Information for package Uyuni-Server-release:
---------------------------------------------
Repository     : Uyuni Server Stable
Name           : Uyuni-Server-release
Version        : 2023.09-230900.207.8.uyuni3
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes
Status         : up-to-date
Source package : Uyuni-Server-release-2023.09-230900.207.8.uyuni3.src
Summary        : Uyuni Server
Description    :
    Uyuni lets you efficiently manage physical, virtual,
    and cloud-based Linux systems. It provides automated and cost-effective
    configuration and software management, asset management, and system
    provisioning.
```

### Uyuni proxy version (if used)

```bash
N/A
```

### Useful logs

```bash
Nothing obvious
```

**Comment by TRISHAHALDER** [comment]
please assign me this issue.

**Comment by avshiliaev** [comment]
@TRISHAHALDER you're free to create a PR if you feel like fixing this :smiley:

**Comment by fullerje** [comment]
@TRISHAHALDER @avshiliaev 

Marking this bug as resolved in the 2023.10 release and all is working normally again.  Thank you!
